File: FishBunDemo/src/main/java/com/sangcomz/fishbundemo/ImageController.java
package com.sangcomz.fishbundemo;

import android.net.Uri;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

/**
 * Created by sangc on 2015-11-06.
 */
class ImageController {
    ImageView imgMain;

    ImageController(ImageView imgMain) {
        this.imgMain = imgMain;
    }

    void setImgMain(Uri path) {
        Picasso
                .get()
                .load(path)
                .fit()
                .centerCrop()
                .into(imgMain);
    }
}

File: FishBunDemo/src/main/java/com/sangcomz/fishbundemo/ImageController.kt
package com.sangcomz.fishbundemo

import android.net.Uri
import android.widget.ImageView
import coil.api.load
import coil.size.Scale

/**
 * Created by sangc on 2015-11-06.
 */
class ImageController(private val imgMain: ImageView) {

    fun setImgMain(path: Uri) {
        imgMain.load(path) {
            scale(Scale.FILL)
            scale(Scale.FIT)
        }
    }
}